lib/bundler/cli/pristine.rb

```diff
@@ -30,6 +30,10 @@ def run
           FileUtils.rm_rf spec.full_gem_path
         when Source::Git
           source.remote!
+          if extension_cache_path = source.extension_cache_path(spec)
```

Comment:
should we be doing this for rubygems sources as well?

Reply:
@segiddins from what I could tell, the extension isn't being cached when installed via RubyGems. I may be mistaken, however, as most of my initial investigation was done on the 1.6 which doesn't have the cache.

Reply:
I believe extension-caching is RubyGems only, so this seems right to me. 👍

Reply:
Ignore me, that comment doesn't actually make sense.